training/distributed_training/pytorch/model_parallel/vit/submit_trainer_vit.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.        image_uri="763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.12.0-gpu-py38-cu113-ubuntu20.04-sagemaker-v1.0",
</pre><p>Use official DLC by specifying framework version, do not use <code>image_uri</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3572/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>